lib/options.json

```diff
@@ -374,6 +374,34 @@
           "enum": ["sockjs", "ws"]
         }
       ]
+    },
+    "watchFiles": {
+      "anyOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "paths": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "options": {
+              "type": "object"
+            }
+          },
+          "additionalProperties": false
+        },
```

Comment:
You can pass multiple objects

Reply:
yep lets change it

Reply:
For better code please look at `definitions`, i.e.`WatchFilesObject` and `WatchFilesString`

Reply:
do we keep single object like `{ paths: , options: }` or enforce array of objects always `{ paths: , options: }[]`?

Reply:
Two things possible, i.e. you can pass multiple paths and you can use multiple objects, make sense because different paths can have different options, for example polling

Reply:
fixed options, wip on test

Reply:
@alexander-akait calling `this.watchFiles` will actually append watchers to `staticWatchers` is that fine or do we need to maintain separate array for our own file watchers? (all are called .close()) when server is terminated

Reply:
Yep, it already works as expected